.github/ISSUE_TEMPLATE.md

```diff
@@ -44,3 +44,6 @@ https://codesandbox.io/s/rkmNRByE4
 ## Expected Behavior
 
 ## Actual Behavior
+
+<!-- Love styled-components? Please consider supporting our collective:
```

Comment:
hmm, should this be added here?

Reply:
Yeah this is the only one that feels weird, the rest I love!